tools/docker/static-analysis/cppcheck.sh

```diff
@@ -21,6 +21,7 @@ run_cppcheck() {
              --force \
              --error-exitcode=1 \
              --enable=warning,style,information \
+             --suppressions-list="$rootdir"/tools/docker/static-analysis/suppressions.txt \
```

Comment:
I don't understand this commit. Please add explanation.

Reply:
Add detailed commit message.